src/documents/views.py

```diff
@@ -1728,7 +1730,7 @@ def get(self, request, format=None):
                     },
                 },
                 "tasks": {
-                    "redis_url": redis_url,
+                    "redis_url": f"{redis_url_parsed.hostname}:{redis_url_parsed.port}",
```

Comment:
For a socket connection, the hostname and port will be empty.

Maybe something like `{.scheme}:{.hostname or .path}`.  That wouldn't include the port though, which is tricky as there may be an extra colon for socket schemes, followed by an empty string.

Reply:
Ah thanks, let me know if youre good with the change, should handle that better.